tests/expected/auto/rgb_u8.json

```diff
@@ -1,4 +1,28 @@
 {
+  "custom_grid": {
```

Comment:
Actually I'm not sure shoud we added them here( It's a little bit ugly..)

Maybe add these fileds to the json of `/catalog`?

Reply:
{
   "fonts": {...},
    "spirtes": {...},
    "tiles": {....},
    "custom": {....}
}

Reply:
Maybe we need to figure out a general way to support tile grid not 3857 in Martin.